core/src/cluster_info.rs

```diff
@@ -151,6 +151,11 @@ impl Signable for PruneData {
     }
 }
 
+struct PullData {
+    pub caller: CrdsValue,
```

Comment:
We don’t need to group by caller. Batching responses is unlikely to add a measurable performance difference since the packets are so small.

Reply:
okay I'll look into processing all available filters simultaneously.